tests/tensor_search/test_get_documents_by_ids.py

```diff
@@ -135,10 +135,10 @@ def test_get_documents_env_limit(self):
                 enums.IndexSettingsField.model: enums.MlModel.bert
             }})
         docs = [{"Title": "a", "_id": uuid.uuid4().__str__()} for _ in range(2000)]
-        tensor_search.add_documents_orchestrator(
-            config=self.config, batch_size=50, add_docs_params=AddDocsParams(
+        tensor_search.add_documents(
+            config=self.config, add_docs_params=AddDocsParams(
                 index_name=self.index_name_2,
-                docs=docs, auto_refresh=False
+                docs=docs, auto_refresh=False, device="cpu"
```

Comment:
Why do we need to manually specify device? I thought that the Pydantic object could determine the default device?

Reply:
solved